app/models/mixins/relationship_mixin.rb

```diff
@@ -673,11 +673,7 @@ def puts_relationship_tree
   def self.deprecate_of_type_parameter(*args)
     return args if args.empty? || args.first.kind_of?(Hash)
 
-    unless Rails.env.production?
-      msg = "[DEPRECATION] of_type parameter without hash symbol is deprecated.  Please use :of_type => 'Type' style instead.  At #{caller[1]}"
-      $log.warn msg
-      warn msg
-    end
+    Vmdb::Deprecation.deprecation_warning("of_type parameter without hash symbol", "use :of_type => 'Type' style instead")
```

Comment:
I think we should warn in production mode if these are methods that we are actually deprecating.  If we don't, users will never know until the method is gone.